sympy/core/tests/test_assumptions.py

```diff
@@ -1134,8 +1134,8 @@ def test_issue_16313():
 def test_issue_16579():
     # complex -> finite | infinite
     # with work on PR 16603 it may be changed in future to complex -> finite
-    x = Symbol('x', complex=True, finite=False)
-    y = Symbol('x', real=True, infinite=False)
+    #x = Symbol('x', complex=True, finite=False)
```

Comment:
This and the comment above can be removed. This whole test function is probably not needed any more although some of the asserts could be moved elsewhere.